teams/3/settings.html

```diff
@@ -0,0 +1,361 @@
+                <!-- This span and svg for icons of nav items -->
+                <span><svg></svg></span>
+                Dashboard
+                <!-- This span for displaying number of notifications -->
+                <span>span</span>
+              </a>
+            </li>
+            <li>
+              <a href="">
+                <!-- This span and svg for icons of nav items -->
+                <span><svg></svg></span>
+                Projects
+                <!-- This span for displaying number of notifications -->
+                <span>span</span>
+              </a>
+            </li>
+            <li>
+              <a href="">
+                <!-- This span and svg for icons of nav items -->
+                <span><svg></svg></span>
+                Tasks
+                <!-- This span for displaying number of notifications -->
+                <span>span</span>
+              </a>
+            </li>
+            <li>
+              <a href="">
+                <!-- This span and svg for icons of nav items -->
+                <span><svg></svg></span>
+                Reporting
+                <!-- This span for displaying number of notifications -->
+                <span>span</span>
+              </a>
+            </li>
+            <li>
+              <a href="">
+                <!-- This span and svg for icons of nav items -->
+                <span><svg></svg></span>
+                Users
+                <!-- This span for displaying number of notifications -->
+                <span>span</span>
+              </a>
+            </li>
+            <li>
+              <a href="">
+                <!-- This span and svg for icons of nav items -->
+                <span><svg></svg></span>
+                Support
+                <!-- This span for displaying number of notifications -->
+                <span>span</span>
+              </a>
+            </li>
+            <li>
+              <a href="">
+                <!-- This span and svg for icons of nav items -->
+                <span><svg></svg></span>
+                Settings
+                <!-- This span for displaying number of notifications -->
+                <span>span</span>
+              </a>
+            </li>
+          </ul>
+        </nav>
+        <div>
+          <div>
+            <h3>New features available!</h3>
+            <p>Check out the new dashboard view. Pages now load faster.</p>
+          </div>
+          <div>
+            <a href="" title="nav image linked somewhere"
+              ><img src="" alt="image picturing something"
+            /></a>
+          </div>
+          <div>
+            <button>Dismiss</button>
+            <button>What’s new?</button>
+          </div>
+        </div>
+        <div>
+          <img src="" alt="image of the user" />
+          <span>Olivia Rhye</span>
+          <span>olivia@untitledui.com</span>
+          <button><i></i></button>
+        </div>
+      </article>
+      <article>
```

Comment:
Please also add more details why do you think so. Article tag wraps whole information about payment and billing address, which is also self-contained and can be reusable for other pages. It contains several sections independent from each other.

Reply:
It should be a section tag because a section wraps logical groups of related content while an article as you said contains self-contained content. In this part of the page, you have reversed the usage of section and article. For example, in a social media app, an individual post can be an article while the whole feed will be a section.